contracts/standard/arbitration/ArbitrableTokens/LockedToken.sol

```diff
@@ -55,14 +55,16 @@ contract LockedToken is MintableToken {
      *  @param _to The address to unlock tokens from.
      */
     function partialUnlock(address _to, uint _maxUnlock) public {
-        if (lastUnlock[_to].add(4 weeks) <= now && amountLocked[_to]!=0) {
-            uint amountOfMonths = now.sub(lastUnlock[_to]) / (4 weeks);
+        // solium-disable-next-line security/no-block-members
```

Comment:
If we update dependency to the latest version of kleros-scripts we can get rid of these comment lines.